be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/ElectionValidator.scala

```diff
@@ -79,7 +79,7 @@ sealed class ElectionValidator(dbActorRef: => AskableActorRef) extends MessageDa
         val electionId: Hash = channel.extractChildChannel
         val sender: PublicKey = message.sender
 
-        val laoId: Hash = channel.decodeChannelLaoId
+        val laoId: Hash = channel.decodeChannelLaoId.get
```

Comment:
are we sure that this will never return None? Otherwise the .get might fail

Reply:
same for below

Reply:
> are we sure that this will never return None? Otherwise the .get might fail

You are right, it could fail